__test__/unit/pushSubscription/nativePermissionChange.test.ts

```diff
@@ -18,7 +19,7 @@ describe('Notification Types are set correctly on subscription change', () => {
     jest.resetAllMocks();
   });
 
-  test('When native permission is rejected, we update notification_types to -2 & enabled to false', async () => {
+  test('When native permission is rejected, we update notification_types to 0 & enabled to false', async () => {
```

Comment:
when the native permission is rejected, the notification types being 0 is equivalent to there not being a permission yet. I still think this should be -2

Reply:
Talked about this outside of this thread, concussion was the names of the enums were misleading. A follow up commit was made to rename them to address this.